dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -49,6 +49,8 @@ repertiore.LOW=Cipher suites without encryption
 repertiore.CUSTOM=Custom list of cipher suites
 repertoire.verifyHostname=Host name verification
 repertoire.verifyHostname.desc=Specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default.
+repertoire.trustDefaultCerts=Trust the default certificates
+repertoire.trustDefaultCerts.desc=Specifies whether to trust the default certificates when establishing trust.  If set to true, then the default certificates be used in addition to the configured trust file to establish trust.  The attribute is set to false by default.
```

Comment:
Change
"Specifies whether to trust the default certificates when establishing trust. If set to true, then the default certificates **be** used in addition to the configured trust file to establish trust."
To
"Specifies whether the trust manager can establish trust by using the default certificates. If set to true, then the default certificates **are** used in addition to the configured trust file to establish trust."